src/EditorFeatures/CSharpTest2/Recommendations/IsKeywordRecommenderTests.cs

```diff
@@ -167,5 +167,37 @@ public class IsKeywordRecommenderTests : KeywordRecommenderTests
             await VerifyAbsenceAsync(AddInsideMethod(
 @"var x = .$$0;"));
         }
+
```

Comment:
Are there sufficient positive test cases to ensure this doesn't cause `as/is` to go missing when they shouldn't?